parachain/pallets/outbound-queue/src/lib.rs

```diff
@@ -338,7 +340,6 @@ pub mod pallet {
 		}
 
 		fn submit(ticket: Self::Ticket) -> Result<MessageHash, SubmitError> {
-			Self::ensure_not_halted().map_err(|_| SubmitError::BridgeHalted)?;
```

Comment:
Removing this check would mean that messages can still be enqueued, even though the bridge is halted. I understand check in `process_message` would prevent the message from being processed, but I am wondering if we shouldn't even enqueue the message in the first place? The current implementation would allow the message to be enqueued, and then not processed in `process_message`. Should we not perhaps leave this check here?